### ISSUE TITLE
Copied the db to Documents folder, still getting errors (no such table)

### ISSUE THREAD

**Issue by AegerBorder** [stephencelis/SQLite.swift#767]
Hi,

I have a prefilled (do you say that in English?) database dropped to my project navigator and I also successfully ran [this code](https://gist.github.com/rd13/320b3fed4d5f10ab8ea833b3f9818980) which copies it to the documents folder. It's an iOS App by the way.
I have a print saying it was successfully copied and as long as I'm running the app in the iOS Simulator the whole thing runs completely without any errors.

But: as soon as I run it on my personal, physical device the print says the database was copied successfully, though, when any button is pressed, I'm getting the error "no such table: filme (code: 1)"
"filme" ("movies" in German) is my table - there's only this one.

I have absolutely no clue what to paste here when it comes to code because it's 1,300 lines long :)
Can you somehow imagine what the problem could be by only reading this?

Thanks for any hints and tipps
Patrick

**Comment by AegerBorder** [comment]
Holy cr*** that's the second time in two days that I post an issue and it solves by itself... more or less.
I was so depressed that nothing worked on the physical device that I deleted the whole app from the iPhone when I thought "Naah... giving up is not a thing". So I reinstalled it and that da*n thing runs perfectly now.

But this leads me to a new thought: will the app run on another physical device which has a previous version installed? That'd not be that good I guess. I'll try. If you're interested in what will happen don't close here :)
